lib/gc/ExecutionEngine/CPURuntime/Microkernel/BrgemmOnednn.cpp

```diff
@@ -137,24 +150,38 @@ void dnnl_brgemm_tilerelease() {
 void dnnl_brgemm_execute(int64_t kernel_idx, void *A, uint64_t A_offset,
                          void *B, uint64_t B_offset, void *C, uint64_t C_offset,
                          int num) {
+  auto it = tl_cache.find(kernel_idx);
```

Comment:
It's better not define tl_cache as global static, define it here as a function static.

Reply:
thanks for the advice, fixed.